### ISSUE TITLE
IOS SLRemoteComposeViewController error.

### ISSUE THREAD

**Issue by jodymac** [OpenFlutter/flutter_share_me#47]
When running on ios device it will say everything worked when sharing to facebook.  I do have facebook installed and it finds my account just fine.  The post never appears.  If I run the device hooked into xcode so I can watch the errors, I get the following after perceiving the post to work just fine.

  2021-08-12 20:05:38.680313-0700 Runner[3385:1487872] [core] SLRemoteComposeViewController: (this may be harmless) viewServiceDidTerminateWithError: Error Domain=_UIViewServiceErrorDomain Code=1 "(null)" UserInfo={Terminated=disconnect method}
2021-08-12 20:05:41.422189-0700 Runner[3385:1491843] [] tcp_input [C20.1:3] flags=[R] seq=850013987, ack=0, win=0 state=CLOSED rcv_nxt=850013987, snd_una=485929286
2021-08-12 20:05:41.422371-0700 Runner[3385:1491843] [] tcp_input [C20.1:3] flags=[R] seq=850013987, ack=0, win=0 state=CLOSED rcv_nxt=850013987, snd_una=485929286

I've updated the pods and tried again.

Installing FBSDKCoreKit (5.15.1)
Installing FBSDKShareKit (5.15.1)
Installing FacebookCore (0.9.0)
Installing FacebookShare (0.9.0)

This was happening on version 0.11.2 and I just updated to 1.0.0 and tried with the same result.

IMPORTANT :  This goes beyond the plugin.  There are issues of this of this on stack overflow.  Fixes are random and all over the place, including updating the facebook SDK's, changing the function call from url to urls, etc...  Not sure which it could be, but I'm very sure it's not you.  Just have to find the workaround.

**Comment by jigarfumakiya** [comment]
Yes, I agree there are issues with this on Facebook SDK as well.
I Will try to find a workaround and inform you.

**Comment by jodymac** [comment]
I've run this off a release build through Testflight with the same results.

**Comment by umesh-mobi** [comment]
Hi,
I am facing the same issue. Could you please provide the solution if you got it previously.
